connect-app/app/screens/Login.jsx

```diff
@@ -0,0 +1,155 @@
+import React, { useEffect, useState } from "react";
+import {
+  Button,
+  KeyboardAvoidingView,
+  StyleSheet,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View,
+} from "react-native";
+import { auth } from "../../FirebaseConfig";
+import {
+  createUserWithEmailAndPassword,
+  signInWithEmailAndPassword,
+} from "firebase/auth";
+import { useNavigation } from "@react-navigation/core";
+
+const Login = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [loading, setloading] = useState(false);
+
+  const navigation = useNavigation();
+
+  useEffect(() => {
+    const unsubscribe = auth.onAuthStateChanged((user) => {
+      if (user) {
+        navigation.replace("Home");
+      }
+    });
+
+    return unsubscribe;
+  }, []);
+
+  const signIn = async () => {
+    setloading(true);
+    try {
+      const response = await signInWithEmailAndPassword(auth, email, password);
+      console.log(response);
+    } catch (error) {
+      console.log(error);
+      alert("Sign in failed: " + error.message);
+    } finally {
+      setloading(false);
+    }
+  };
+
+  const signUp = async () => {
+    setloading(true);
+    try {
+      const response = await createUserWithEmailAndPassword(
+        auth,
+        email,
+        password
+      );
+      console.log(response);
+      alert("Check your email");
```

Comment:
Why are we using this alert? Is it because user receives any email like confirmation link? or activation link?

Reply:
```suggestion
      alert("Check your email");
```
if it's not sending any link then can we alert with login successful message..

Reply:
Aah I missed this! This alert is useless right now, but when signUp is successful, instead of alert, we should add a functionality to send a mail to the user's email with body "Account has been created successfully" 
This is a To-Do item to consider. Let me know your thoughts on this!

Reply:
I believe this part should be in the backend as well.